src/routes/safe/components/Transactions/TxsTable/ExpandedTx/OwnersColumn/index.tsx

```diff
@@ -143,6 +143,7 @@ const OwnersColumn = ({
     displayButtonRow = false
   }
 
+  // TODO: simplify this whole logic around tx status, it's getting hard to maintain and follow
```

Comment:
would be great if you create a ticket for this so it doesn't get lost in the source code

Reply:
https://app.zenhub.com/workspaces/safe-multisig---web-5ce554debb310a35b2f8b6f8/issues/gnosis/safe-react/1339